src/Modules/BuildList.lua

```diff
@@ -60,9 +60,14 @@ function listMode:Init(selBuildName, subPath)
 		self:SortList()
 	end)
 	self.controls.sort:SelByValue(main.buildSortMode, "sortMode")
-	self.controls.buildList = new("BuildListControl", {"TOP",self.anchor,"TOP"}, 0, 50, 640, 0, self)
+	self.controls.buildList = new("BuildListControl", {"TOP",self.anchor,"TOP"}, 0, 75, 640, 0, self)
 	self.controls.buildList.height = function()
-		return main.screenH - 58
+		return main.screenH - 80
+	end
+	self.controls.searchText = new("EditControl", {"TOP",self.anchor,"TOP"}, 0, 25, 640, 20, self.filterBuildList, "Search", "%c%(%)", 100)
+	self.controls.searchText.enterFunc = function(buf)
+		main.filterBuildList = buf
+		self:BuildList()
 	end
```

Comment:
You can also do it like this to use a `changeFunc` instead of `enterFunc`.  This does run `NewFileSearch` for every keypress, though, which obviously can feel a bit more laggy.  What you could do here to minimize that lag is instead of having this function call `self:BuildList()`, it could create the list with all builds, loop through that list in this changeFunc, and build a filtered list that it assigns to `self.list` for the build list to use.  

That latter bit could be a separate PR, but if you could do it here that'd be great.  I do think using a `changeFunc` instead of `enterFunc` is more consistent with other lists, though.

```suggestion
	self.controls.searchText = new("EditControl", {"TOP",self.anchor,"TOP"}, 0, 25, 640, 20, self.filterBuildList, "Search", "%c%(%)", 100, function(buf)
		main.filterBuildList = buf
		self:BuildList()
	end)
```

Reply:
I created a new issue #2773 for this change, as it's a small QoL bonus.  I would've done the `changeFunc` bit myself, but it feels too laggy to be useful, and I don't have the time atm to re-implement how this list works internally.